### DISCUSSION TITLE
「开源之史」系列之六/四：直面人性的暗面——致软件爱好者公开信始末 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#48] [Announcements]
# 「开源之史」系列之六/四：直面人性的暗面——致软件爱好者公开信始末 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/02-03-001-ip-law-and-license-evolution-microsoft-story/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

现在想来, Bill 当年的怒吼, 除了表演因素之外,
其实并没多少阴暗面, 有点儿象, 初次到美洲发现 土豆 时, 多数海员本身文化不高,
完全意志不到这是人类一次重大飞跃的入口;
只有拥有足够视野/能力/经验/...以及想象力的人,
才可能意识到 土豆 可以形成多么广大的市场, 并立即展开行动, 注册商标/专利/...垄断世界淀粉行业200年...

正如 Bill 发现代码和版权的天然关联, 并创造出软件许可证;
这当然平地创造出一个前所未有的软件市场来,
同时, 也给 RMS 一个巨大的灵感,
用相同思路, 构造出 GPL 来重新令软件自由...

所以, 历史上发生的一切, 都是最正确的选择, 
无论被什么逼的, 总之那一瞬间, 所有人都爆发出自身当时 MAX 的创造力...

现在轮到我们了...

**Reply by lijiangsheng1** [reply]
我们需要从创作者的角度出发才有可能被激发，如果仍然延续偷盗（就是bill 说的盗窃），则永无出头之日。
